### DISCUSSION TITLE
Is there a way to view commit history reversely?

### DISCUSSION THREAD

**Discussion by zoubingwu** [gitkraken/vscode-gitlens#2562] [Q&A]
Sometimes I want to see how a project evolves from the start. It can be done with `git log --reverse` then checkout to that commit, but it will be good to be able to do that in the UI for ease of use.

The commit graph is a good place but I can't find a way to sort commit by date.

**Comment by jkelroy** [comment]
Hello @zoubingwu, there is no setting to reverse the commit order. However, you can consider submitting a feature request to support this [here](https://github.com/gitkraken/vscode-gitlens/issues/new/choose).
